### ISSUE TITLE
Connections - Jellyfin "Update library" not working

### ISSUE THREAD

**Issue by pimw1** [Radarr/Radarr#10197]
### Is there an existing issue for this?

- [X] I have searched the existing open and closed issues

### Current Behavior

Hi,

I've setup a connection to Jellyfin, according to the below settings
![image](https://github.com/user-attachments/assets/49adaa34-7c28-4e6a-b834-715e646d9003)

As you can see, i've enabled "Update library"  and disabled "Send notificiations".

After triggering the trigger (in my case: deleting a movie (file)), an unsuccessful attempt is made to send a notification. The Jellyfin library is not updated.

The below warning is visible:
|Warn|HttpClient|HTTP Error - Res: HTTP/1.1 [POST] http://192.168.178.104:8096/mediabrowser/Notifications/Admin: 404.NotFound (0 bytes)

Traces, debug log and logs are provided.

### Expected Behavior

After triggering the trigger (in my case: deleting a movie (file)), i would expect an update of the Jellyfin library and no attempt to send notifications.

### Environment

```markdown
- OS: openmediavault 7.3.0-3 (Sandworm)
- Radarr: 5.8.3.8933
- Docker Install: Yes
- Using Reverse Proxy: No 
- Browser: Chrome
- Database: i don't know, it is the default database from this docker setup:

  radarr:
    container_name: radarr
    image: lscr.io/linuxserver/radarr:latest
    networks:
      - mynetwork
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Amsterdam
    ports:
      - 7878:7878
    volumes:
      - ${radarr_config}:/config
      - ${downloads}:/downloads
    restart: unless-stopped
```

### What branch are you running?

Master

### Trace Logs? **Not Optional**

[radarr.txt](https://github.com/user-attachments/files/16347917/radarr.txt)
[radarr.debug.txt](https://github.com/user-attachments/files/16347918/radarr.debug.txt)
[radarr.trace.txt](https://github.com/user-attachments/files/16347919/radarr.trace.txt)

### Trace Logs have been provided as applicable. Reports will be closed if the required logs are not provided.

- [X] I have read and followed the steps in the wiki link above and provided the required trace logs - the logs contain `trace` - that are relevant and show this issue.

**Comment by pimw1** [comment]
Hi, thank you for doing the triage so quickly. I'm not sure about the conclusion though: the incident shows an issue in which the "do not send notification" is ignored within Radarr. That seems to be a bug to me?

**Comment by pimw1** [comment]
Thank you for replying. I've noticed that issue before. Quote from that issue: 

> TL;DR; You'll want to uncheck Send Notifications and leave Update Library enabled

As can be seen in the image, i've unchecked "Send notifications" and have enabeld "Update library". The problem is that it still tries to send a notification. Therefore, this behaviour is different than desribed in #10026 (unless i am misinterpreting something??).

**Comment by pimw1** [comment]
Thanks to help from discord, the root cause is discovered: the absence of unified path definitions between the jellyfin docker container and the radarr docker container. This was a configuration mistake from my side. I've corrected this and everything is working again. 

Thanks everyone for the help.
